### DISCUSSION TITLE
how to use the mouse to extract pointcloud coordinates by python?

### DISCUSSION THREAD

**Discussion by JLUJYP** [isl-org/Open3D#4568] [Q&A]
I want to get   pointcloud coordinates by clicking the mouse. how to achieve this with Open3D? Thank you very much!

**Comment by errissa** [comment]
There is an example that shows exactly how to do that: `examples/python/gui/mouse-and-point-coord.py`.

**Reply by JLUJYP** [reply]
Thank you so much! The problem was solved!
